web-client/src/components/StepTracker/StepTracker.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { COLORS } from 'src/theme/colors';
+import styled from 'styled-components';
+
+const getStatus = (idx, current) => {
```

Comment:
nit: move below StepTracker component so top-level component is first in the file